vaadin-notification-flow-demo/src/main/java/com/vaadin/flow/component/notification/demo/NotificationView.java

```diff
@@ -141,4 +143,41 @@ private void addStyledNotificationContent() {
         button.setId("styled-content-notification-button");
         addCard("Notification with styled content", button);
     }
+
+    private void createThemeVariants() {
+
```

Comment:
Nit: extra line.

Reply:
The Vaadin Java conventions allow an empty line in the beginning of the method. Personally I like it sometimes to have additional spacing. Anyway, removed.